test/e2e/common/container_probe.go

```diff
@@ -390,7 +392,8 @@ var _ = framework.KubeDescribe("Probing container", func() {
 				},
 			},
 			InitialDelaySeconds: 0,
-			FailureThreshold:    60,
+			PeriodSeconds:       1,
```

Comment:
hm, 10 seconds is not fast enough given there are 45 seconds to startup? Looks fishy

Reply:
yes, but I think it depends on the load as well
we check the time difference between **pod** Started and **container** Ready
note that **pod** Started means the containers have everything to start, which is different from **container** Started which means the container runs AND the `startupProbe` has succeeded
we should be checking the time between the latter and Ready state